src/app/modules/main/module.nim

```diff
@@ -767,13 +767,11 @@ method getVerificationRequestFrom*[T](self: Module[T], publicKey: string): Verif
 
 method getContactDetailsAsJson*[T](self: Module[T], publicKey: string, getVerificationRequest: bool): string =
   let contact = self.controller.getContact(publicKey)
-  let (name, _, _) = self.controller.getContactNameAndImage(contact.id)
   var requestStatus = 0
   if getVerificationRequest:
     requestStatus = self.getVerificationRequestFrom(publicKey).status.int
   let jsonObj = %* {
-    "displayName": name,
-    "optionalName": self.controller.getContactDetails(contact.id).optionalName, # original display name, if renamed
```

Comment:
We still need this in the community user list panel, to be able to display the _original_ display name below the local nickname

Reply:
"optionalName" was used in one place only: ProfileDialogView. I removed usage of getContactDetailsAsJson there so this field is not needed anymore.

Reply:
OK, thx